meta-freescale-3rdparty-mbl/conf/machine/imx6ul-pico-mbl.conf

```diff
@@ -100,6 +100,9 @@ BBMASK += "/meta-fsl-bsp-release/imx/meta-bsp/classes/(?!fsl-eula-unpack\.bbclas
 BBMASK += "/meta-freescale/classes/fsl-eula-unpack.bbclass"
 BBMASK += "/meta-mbl/meta-fsl-bsp-release-mbl/imx/meta-bsp/recipes-bsp/firmware-imx/"
 
+# Remove all conflicting meta-mbl-bsp-common linux-* recipes.
+BBMASK += "/meta-mbl/meta-mbl-bsp-common/recipes-kernel/linux/linux-raspberrypi_%.bbappend"
```

Comment:
I don't get why this, and similar lines in the other machine confs are necessary. Why does it matter to imx6ul-pico-mbl if the linux-raspberrypi recipe is appended?

Reply:
If this BBMASK line is omitted then bitbake mbl-image-development recipe parsing generated and error reporting there is no linux-raspberrypi_%.bb recipe, for example.

Reply:
Ah, okay. I think I agree with @diego-sueiro's comment below:

> You don't need to add these `bbappends` to be able to `require linux-mbl.inc`,
You can do this directly in the _originals_ `bbappends` and get rid of these BBMASK in the `${MACHINE}.conf` files.

Reply:
I agree. Done in 49d92b95b12ecb1b66bb1f6dc794646d4c3d707d.

Reply:
Thanks.